Makefile

```diff
@@ -44,14 +52,31 @@ export GOARCH ?= amd64
 
 
 # @todo allow user to run for a single $PKG only?
-PACKAGES := $(shell $(GO) list ./...)
+PACKAGES_CMD := $(GO) list ./...
 GO_EXCLUDE := /vendor/|.pb.go|.gen.go
-GO_FILES := $(shell find . -name '*.go' | grep -v -E '$(GO_EXCLUDE)')
+GO_FILES_CMD := find . -name '*.go' | grep -v -E '$(GO_EXCLUDE)'
```

Comment:
shouldn't this be inside "" ? isn't the | triggered ?

Reply:
"Triggered" meaning evaluated while determining the value for GO_FILES_CMD or causing grep to be executed?  I'd expect make to do variable substitution but otherwise treat the line as a string.  I did a quick test using an empty path on a short makefile like the following and "make foo" didn't try to execute find or grep until it was trying to execute the recipe for foo:

GO_EXCLUDE := /vendor/|.pb.go|.gen.go
GO_FILES_CMD := find . -name '*.go' | grep -v -E '$(GO_EXCLUDE)'

foo:
	/bin/echo starting rule eval for foo
	$(GO_FILES_CMD)
	/bin/echo ending rule eval for foo